include/mpi_shared_linear_elasticity.h

```diff
@@ -55,6 +55,10 @@ namespace Solid
        * Assembles lhs and rhs. At time step 0, the lhs is the mass matrix;
        * at all the following steps, it is \f$ M + \beta{\Delta{t}}^2K \f$.
        */
+      using SharedSolidSolver<dim>::cellwise_sxx;
```

Comment:
These should go to ``inheritance_macros.h``

Reply:
Changed.

Reply:
Fixed